### ISSUE TITLE
[A11Y] Unable to close “Learn more”/“What is a web compatibility issue”  popups using “Esc” key

### ISSUE THREAD

**Issue by softvision-oana-arbuzov** [webcompat/webcompat.com#3397]
**Browser / Version**: Firefox Nightly 80.0a1 (2020-07-15)
**Operating System**: Windows 10 Pro

**Steps to Reproduce**:
1. Navigate to https://webcompat.com/issues/new
2. Using "Tab" key, navigate to the URL field and fill it in. 
3. Using "Tab" key, navigate to "Confirm" button.
4. Using "Tab" key, navigate to "Learn more about web compatibility" link.
5. Using "Enter" key, activate the popup.
6. Using "Esc" key, try to dismiss the popup.
7. Observe behavior.

**Expected Behavior:**
The popup is  dismissed.

**Actual Behavior:**
Nothing happens, the popup is not dismissed.

**Notes:**
1. Screenshot attached.
2. The issue is also reproducible for "Learn more" > "Public Reports" popup.
![image](https://user-images.githubusercontent.com/12184325/87690248-dff85280-c791-11ea-935d-101a20dbedce.png)
![image](https://user-images.githubusercontent.com/12184325/87690317-f9999a00-c791-11ea-9a31-3021e7f7032f.png)
3. According to "Inspector>Accessibility" tab, "X" button "must be focusable and should have interactive semantics".
![image](https://user-images.githubusercontent.com/12184325/88665372-32693580-d0e7-11ea-9676-ed59103002fc.png)

Watchers:
@softvision-oana-arbuzov
@cipriansv

**Comment by softvision-oana-arbuzov** [comment]
I can confirm this is fixed.

**Tested with:**
Browser / Version: Firefox Nightly 82.0a1 (2020-09-16), Chrome 85.0.4183.102
Operating System: Windows 10 Pro
